tests/topotests/isis-topo1-vrf/r1/isisd.conf

```diff
@@ -0,0 +1,15 @@
+hostname r1
```

Comment:
Please put the new topotest on a separate commit.

Reply:
Sure, will do it separate commit. 
Thanks .